index.js

```diff
@@ -74,7 +75,11 @@ var initConfig = function(projectConfig) {
   util._extend(config, require('./config.js').trace);
   util._extend(config, projectConfig);
   if (process.env.hasOwnProperty('GCLOUD_TRACE_LOGLEVEL')) {
-    config.logLevel = process.env.GCLOUD_TRACE_LOGLEVEL;
+    try {
+      config.logLevel = parseInt(process.env.GCLOUD_TRACE_LOGLEVEL);
+    } catch (e) {
+      // Just use the original log level if GCLOUD_TRACE_LOGLEVEL is malformed.
```

Comment:
Makes sense - though the logger isn't initialized at this point (since it depends on the value of `config.logLevel`).

Do you think it'd make sense to defer printing of the warning until after the logger is initialized?